.github/workflows/build-templates.yml

```diff
@@ -110,6 +94,8 @@ jobs:
           path: |
             ${{ env.work_dir }}/**/node_modules
           key: ${{ runner.os }}-library-yarn-${{ hashFiles(format('{0}/**/package.json', env.work_dir)) }}
+          restore-keys: |
+            ${{ runner.os }}-library-yarn-
```

Comment:
Is this `-` at the end necessary?

Reply:
@MateWW not in this specific case, but generally it's good to add it to avoid matching words that start with these letters so I generally follow this practice

Reply:
Oki, Thanks for explaination!